rta/evasion_ntdll_from_unusual_path.py

```diff
@@ -0,0 +1,37 @@
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License
+# 2.0; you may not use this file except in compliance with the Elastic License
+# 2.0.
+
+
+from . import common
+from . import RtaMetadata
+import win32file, win32api, os
+from os import path
+
+metadata = RtaMetadata(
+    uuid="e6d5315f-4c70-4788-8564-e7c23786a4d0",
+    platforms=["windows"],
+    endpoint=[{"rule_name": "NTDLL Loaded from an Unusual Path", "rule_id": "3205274e-7eb0-4765-a712-5783361091ae"}],    
+    siem=[],
+    techniques=["T1055"],
+)
+
+@common.requires_os(metadata.platforms)
+
+
```

Comment:
```suggestion
@common.requires_os(*metadata.platforms)
```

every where this syntax is used, it needs an asterisk `*metadata.platforms`.

I realize this may be an existing bug across existing RTAs

Reply:
#2871